app/assets/javascripts/models/ui/state_manager.coffee

```diff
@@ -177,7 +187,10 @@
       component = components.shift()
 
       state.visible = component != 'map'
-      state.parent = !component
+      if component == 'collections'
+        state.parent = false
+      else
+        state.parent = state.manualShowParent
```

Comment:
Shouldn't these 4 lines be something like:

    state.parent = !component && state.manualShowParent